### PR TITLE
RuboCop: Fix Layout/IndentHash

### PR DESCRIPTION
**Pull request by leila-alderman**
This fixes the RuboCop to-do for the indentation of the first key inside
of a hash literal where the brace and the first key are on separate
lines. To better align with the existing code conventions, this cop was
set to use the `consistent` style instead of the default
`special_inside_parentheses` style.

All unit tests:
4444 tests, 71479 assertions, 0 failures, 0 errors, 0 pendings, 2
omissions, 0 notifications
100% passed

### CODE REVIEW

File: lib/active_merchant/billing/gateways/authorize_net.rb [hunk 1]
```diff
@@ -85,8 +85,8 @@ class AuthorizeNetGateway < Gateway
       AVS_REASON_CODES = %w(27 45)
 
       TRACKS = {
-          1 => /^%(?<format_code>.)(?<pan>[\d]{1,19}+)\^(?<name>.{2,26})\^(?<expiration>[\d]{0,4}|\^)(?<service_code>[\d]{0,3}|\^)(?<discretionary_data>.*)\?\Z/,
-          2 => /\A;(?<pan>[\d]{1,19}+)=(?<expiration>[\d]{0,4}|=)(?<service_code>[\d]{0,3}|=)(?<discretionary_data>.*)\?\Z/
+        1 => /^%(?<format_code>.)(?<pan>[\d]{1,19}+)\^(?<name>.{2,26})\^(?<expiration>[\d]{0,4}|\^)(?<service_code>[\d]{0,3}|\^)(?<discretionary_data>.*)\?\Z/,
```
Comment (fatcatt316) [inline lib/active_merchant/billing/gateways/authorize_net.rb]: 🙌 hooray for 2-space indentation!

File: .rubocop.yml [hunk 2]
```diff
@@ -32,3 +32,6 @@ Layout/DotPosition:
 
 Layout/CaseIndentation:
   EnforcedStyle: end
+
+Layout/IndentHash:
+  EnforcedStyle: consistent
```
Comment (fatcatt316) [inline .rubocop.yml]: What do you think about ending this file with a newline? You can set your editor to do this automagically, too. [Here's some background info about this](https://thoughtbot.com/blog/no-newline-at-end-of-file).

File: test/unit/gateways/clearhaus_test.rb [hunk 3]
```diff
@@ -389,20 +389,20 @@ def failed_authorize_response
...
-            'authorization' => {
-                'href' => '/authorizations/84412a34-fa29-4369-a098-0165a80e8fda'
-            },
-            'refunds' => {
-                'href' => '/authorizations/84412a34-fa29-4369-a098-0165a80e8fda/refunds'
-            }
-        }
+          'refunds' => {
+            'href' => '/authorizations/84412a34-fa29-4369-a098-0165a80e8fda/refunds'
+          }
```
Comment (fatcatt316) [inline test/unit/gateways/clearhaus_test.rb]: The spacing between this line and the next makes it look like there's a missing `}`. Curious why that is?

File: test/unit/gateways/fat_zebra_test.rb [hunk 4]
```diff
@@ -319,28 +319,28 @@ def declined_purchase_response
     {
```
Comment (fatcatt316) [inline test/unit/gateways/fat_zebra_test.rb]: This gateway has an amazing name.

File: .rubocop.yml (hunk 2 above)
Comment (leila-alderman) [reply .rubocop.yml]: So weird that RuboCop didn't complain to me about this...

Comment (fatcatt316) [reply .rubocop.yml]: RuboCop going rogue 😬

File: test/unit/gateways/clearhaus_test.rb (hunk 3 above)
Comment (leila-alderman) [reply test/unit/gateways/clearhaus_test.rb]: Thanks for catching this issue! It turns out that all three of these lines needed to be moved to the left by one tab.

Comment (fatcatt316) [reply test/unit/gateways/clearhaus_test.rb]: I had to run JoeboCop to find it

**Comment by leila-alderman** [comment]
@fatcatt316 Thanks for reviewing this! Since these are just indentation changes, I feel pretty good about not running the remote test files. I attempted to do some spot checking on the remote test files, but I got so many errors that it didn't seem very informative or worthwhile.

File: .rubocop_todo.yml [hunk 5]
```diff
@@ -977,3 +970,4 @@ Style/ZeroLengthPredicate:
 # URISchemes: http, https
 Metrics/LineLength:
   Max: 2602
+
```
Comment (fatcatt316) [inline .rubocop_todo.yml]: 🙌

**Pull request merged**